### PR TITLE
Publisher app migration to carbon-apim 6.5.x

### PR DESCRIPTION
**Pull request by tmkasun**
## Purpose
- Add tempory login service
- Fix ESLint issues
- Git ignore and remove tracking public/dist/ directories both in publisher and store (that's why -27476 :smile: ) 
- Improve `TokenMergeInterceptor`
- Add fetch interceptor to change request Accept header

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "Content-Type": "application/json"
+    }, "json");
+
+    //curl -k -d "grant_type=password&username=admin&password=admin&scope=apim:subscribe" -H "Authorization: Basic SGZFbDFqSlBkZzV0YnRyeGhBd3liTjA1UUdvYTpsNmMwYW9MY1dSM2Z3ZXpIaGM3WG9HT2h0NUFh" https://localhost:8243/token
+
+    var clientId = result.data.clientId;
+    var clientSecret = result.data.clientSecret;
+    var Base64 = org.apache.axiom.om.util.Base64;
+    var String = Packages.java.lang.String;
+    var base64encoded = Base64.encode(new String(clientId+":"+clientSecret).getBytes());
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag]: need to format

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag]: @isharac , Can you please point out which line needs formatting?

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag]: I think it's the spacing fixed with this commit https://github.com/wso2/carbon-apimgt/pull/6209/commits/102d37edcc4cdbc76b8d9d391d21e164dbafb19f

Comment (isharac) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag]: Yes, it's spacing but it should be like
Base64.encode(new String(clientId + ":" + clientSecret).getBytes());

Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/app.jag]: Got it , Fixed with this commit https://github.com/wso2/carbon-apimgt/pull/6209/commits/5a924a6f63d22fca6bcb861b5a1669876a21148a

**Pull request merged**